src/30_Advanced/Scrollbound_Effects_and_Animations.html

```diff
@@ -0,0 +1,353 @@
+
+  <script async custom-element="amp-fit-text" src="https://cdn.ampproject.org/v0/amp-fit-text-0.1.js"></script>
+  <script async custom-element="amp-carousel" src="https://cdn.ampproject.org/v0/amp-carousel-0.1.js"></script>
+
+
+  <link rel="canonical" href="<%host%>/components/amp-position-observer/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+
+  <!--
+    ## Styles
+
+    The CSS used for these samples are included here for reference.
+
+    These rules are simply needed to make the samples work but are not fundamental
+    to the concepts covered here.
+  -->
+  <style amp-custom>
+
+    /*
+     * Fidget Spinner styles
+     */
+    .fidget-spinner-scene {
+      margin: 10px 20%;
+    }
+
+    /*
+     * Card transition styles
+     */
+    .card {
+      margin: 10%;
+      position: relative;
+    }
+    .card-title {
+      padding: 5%;
+      padding-right: 40%;
+      font-size: 50%;
+      font-weight: bold;
+      color: #FAFAFA;
+      transform: translateX(-100%);
+    }
+
+    /*
+     * Parallax window stles
+     */
+    .parallax-image-window {
+      overflow: hidden;
+    }
+    .parallax-image-window amp-img {
+      margin-bottom: -30%;
+    }
+
+    /*
+     * Carousel transition styles
+     */
+    .carousel-container {
+      margin-right: -60%;
+      margin-left: 60%;
+    }
+  </style>
+</head>
+
+<body>
+
+  <!--
+  ## Scrollbound Animations
+
+  Let's create a fidget spinner that rotates as user scrolls the page.
+
+  This sample showcases the core concept behind combining
+  `amp-position-observer` and `amp-animation`: The ability to progress
+  through a keyframe animation timeline as an element progresses through the viewport via scrolling.
+  -->
+
+  <!--
+  We define our fidget spinner scene as a `div` with an `image`.
+
+  We add an `amp-position-observer` element as child of the scene to monitor
+  its progress through the viewport (position observer acts on its immediate parent by default).
+
+  By defining an `on:scroll` action, we seek through the timeline of the keyframe
+  animation that we will define later in the next section.
```

Comment:
Can you mention what `.seekTo(percent=event.percent)` does?

Reply:
restructured and added more details as bullet points.